pkg/mover/chart.go

```diff
@@ -230,15 +224,65 @@ func (cm *ChartMover) Print() {
 	cm.printMove()
 }
 
-func (cm *ChartMover) loadChart(chartPath string) error {
+func (cm *ChartMover) loadChart(src *Source) error {
 	var err error
+	if src.Chart.Local != nil {
+		cm.chart, err = loader.Load(src.Chart.Local.Path)
```

Comment:
Could we maybe add a method in `cm` similar to `cm.loadChartFromIntermediateBundle` so we are consistent?

it's signature could be `cm.LoadChartFromPath(chartPath string) error`

That way you could do `return cm.LoadChartFromPath(src.Chart.Local.Path)`

Reply:
yep, will fix.

Reply:
fixed